src/test/java/com/google/api/codegen/testdata/fakebilling_gapic.yaml

```diff
@@ -20,11 +23,122 @@ license_header:
+    max_rpc_timeout_millis: 60000 # 60 seconds
+    total_timeout_millis: 600000 # 10 minutes
+  - name: messaging
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
+    initial_rpc_timeout_millis: 12000 # 12 seconds
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 12000 # 12 seconds
+    total_timeout_millis: 600000 # 10 minutes
   methods:
-  - name: List
-  - name: Get
+  - name: fakebilling.billingAccounts.list
+    request_object_method: true
+    resource_name_treatment: STATIC_TYPES
+    page_streaming:
+      request:
+        page_size_field: page_size
+        token_field: page_token
+      response:
+        token_field: next_page_token
+        resources_field: ListBillingAccountsResponse
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: fakebilling.billingAccounts.get
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
 - name: google.fakebilling.v1.Projects
+  lang_doc:
+    java: Configure billing projects.
+  collections:
+  - name_pattern: billingAccounts/{billingAccountsId}
+    entity_name: billingAccountsId
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  - name: pull
+    retry_codes:
+    - DEADLINE_EXCEEDED
+    - INTERNAL
+    - CANCELLED
+    - RESOURCE_EXHAUSTED
+    - UNAVAILABLE
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
+    initial_rpc_timeout_millis: 60000 # 60 seconds
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000 # 60 seconds
+    total_timeout_millis: 600000 # 10 minutes
+  - name: messaging
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
+    initial_rpc_timeout_millis: 12000 # 12 seconds
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 12000 # 12 seconds
+    total_timeout_millis: 600000 # 10 minutes
   methods:
-  - name: UpdateBillingInfo
-  - name: GetBillingInfo
-  - name: BeginTransaction
+  - name: fakebilling.projects.updateBillingInfo
+    required_fields:
+      - name
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+    request_object_method: true
+    resource_name_treatment: STATIC_TYPES
+    field_name_patterns:
+      name: name
+  - name: fakebilling.projects.getBillingInfo
+    required_fields:
+      - name
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+    request_object_method: true
+    resource_name_treatment: STATIC_TYPES
+    field_name_patterns:
+      name: name
```

Comment:
I think the value side of this needs to match an `entity_name` in the `collections` section